src/llmcompressor/modifiers/smoothquant/base.py

```diff
@@ -13,6 +13,11 @@
 
 MINIMUM_SMOOTHING_SCALE = 1e-5
 
+DEFAULT_SMOOTHQUANT_MAPPINGS = [
```

Comment:
QQ: if the model does not have the right structure, will we fail loudly?

Reply:
Yes, if it can't find a particular layer it'll fail like this:

```
Could not find targets ['re:.*does_not_exist'] in module LlamaForCausalLM(
```

It won't explicitly state that the error is due to the SmoothQuant defaults but should be sufficient for debugging